templates/toml.j2

```diff
@@ -3,6 +3,8 @@
 {% macro render_option(key, value) %}
 {%   if (value is iterable and ((value is not string) and (value is not mapping))) %}
 {{ key }}={{ value }}
+{%   elif (value is boolean and ((value is not string) and (value is not mapping)))  %}
```

Comment:
Thanks.  Two comments

* this should only check for `value is boolean` because if the value is a boolean, it is definitely not a string and not a mapping (nor any other type)
* this should be the first check in `render_option`

Reply:
Hello, thanks for the feedback, I believe everything should be updated now.